apps/rush-lib/typings/encode-registry/encode-registry.d.ts

```diff
@@ -0,0 +1,6 @@
+// typings for encode-registry@1.0.0
+
+declare module 'encode-registry' {
```

Comment:
>encode [](start = 16, length = 6)

This package was coded in TypeScript -- why do we need typings?

[https://github.com/pnpm/encode-registry/blob/master/package.json#L6](https://github.com/pnpm/encode-registry/blob/master/package.json#L6)